### PR TITLE
maiden 1.0 - docs update

### PR DESCRIPTION
**Pull request by dndrks**
updated screenshots and copy to match new capabilities of maiden 1.0

### CODE REVIEW

**Comment by dndrks** [comment]
@ngwese : could you specifically weigh in on the `CONTRIBUTIONS` section, to help guide folks on best practice for submitting to the community project repo?

File: norns/index.md [hunk 1]
```diff
@@ -272,6 +282,56 @@ If you need to restart the matron/crone environment for any reason (ie, the menu
...
+      "project_name": "NAME",
+      "project_url": "URL",
+      "author": "NAME",
+      "description": "WORDS",
+      "discussion_url": "LINES_LINK",
+      "tags": ["TAG", "TAG", "TAG"],
+      "origin": "IF_APPLICABLE: lines"
+    },
+```
+
```
Comment (pq) [inline norns/index.md:334]: Aside: maybe consider creating a pull request template w/ this boilerplate filled in?

See: https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository

**Review by pq — APPROVED** [review]
Nice docs!

💎 💎 💎

File: norns/index.md [hunk 2]
```diff
@@ -272,6 +282,56 @@ If you need to restart the matron/crone environment for any reason (ie, the menu
...
+This tab shows which projects are currently installed on your norns.
+
+![](image/maiden-installed.png)
+
+Each entry has two actions: **UPDATE** and **REMOVE**.
+
+If you choose to update a project that currently lives on your norns, please note that local modifications you have made will be overwritten. If you wish to retain multiple versions of a project, please reference the [FILE MANAGEMENT](#file-management) section.
+
+Once you update a project through PROJECT MANAGER, you'll see a commit number listed on the right of the project's tile (like *34d225b*). Click a project's commit number to be brought to the project's GitHub page, where you can learn more about the project and verify that the version you have is the latest.
```
Comment (pq) [inline norns/index.md:303]: Consider:

> through the PROJECT MANAGER

(add **_the_**)

File: norns/index.md (hunk 1 above)
Comment (dndrks) [reply norns/index.md:334]: @pq , this is a DOPE idea. brining visibility to @ngwese

**Pull request merged**